tests/OSPSuite.Infrastructure.Tests/Import/concern_for_DataSetToDataRepositoryMapperSpecs.cs

```diff
@@ -0,0 +1,130 @@
+               { 
+                  Column = new Column() 
+                  {
+                     Name = "Time",
+                     Unit = new UnitDescription("s")
+                  },
+                  ColumnInfo = new ColumnInfo()
+                  {
+                     BaseGridName = null,
+                     Name = "Time"
+                  }
+               }, 
+               new List<SimulationPoint>() 
+               {
+                  new SimulationPoint()
+                  {
+                     Unit = "s",
+                     Measurement = 0,
+                     Lloq = double.NaN
+                  },
+                  new SimulationPoint()
+                  {
+                     Unit = "s",
+                     Measurement = 1,
+                     Lloq = double.NaN
+                  },
+                  new SimulationPoint()
+                  {
+                     Unit = "s",
+                     Measurement = 2,
+                     Lloq = double.NaN
+                  }
+               } 
+            },
+            {
+               new ExtendedColumn()
+               {
+                  Column = new Column()
+                  {
+                     Name = "Concentration",
+                     Unit = new UnitDescription("mol")
+                  },
+                  ColumnInfo = new ColumnInfo()
+                  {
+                     BaseGridName = "Time",
+                     Name = "Concentration"
+                  }
+               },
+               new List<SimulationPoint>()
+               {
+                  new SimulationPoint()
+                  {
+                     Unit = "mol",
+                     Measurement = 10,
+                     Lloq = 1
+                  },
+                  new SimulationPoint()
+                  {
+                     Unit = "mol",
+                     Measurement = 0.1,
+                     Lloq = 1
+                  },
+                  new SimulationPoint()
+                  {
+                     Unit = "mol",
+                     Measurement = double.NaN,
+                     Lloq = 1
+                  }
+               }
+            }
+         };
+         dataSet.AddData(new List<ParsedDataSet>() 
+         {
+            new ParsedDataSet(new List<(string, IList<string>)>(), A.Fake<IUnformattedData>(), new List<UnformattedRow>(), parsedData)
+         });
+         cache.Add("sheet1", dataSet);
+         A.CallTo(() => dataSource.DataSets).Returns(cache);
+         var dimensionFactory = A.Fake<IDimensionFactory>();
+         A.CallTo(() => dimensionFactory.DimensionForUnit(A<string>.Ignored)).Returns(Constants.Dimension.NO_DIMENSION);
+         sut = new DataSetToDataRepositoryMapper(dimensionFactory);
+      }
+   }
+
+   public class When_mapping_a_data_repository : concern_for_DataSetToDataRepositoryMapperSpecs
+   {
+      protected override void Because()
+      {
+         base.Because();
```

Comment:
base.Becasue does not make sense.
You are testing a SINGLE Action in your test. If there is something happening in base.because, it should be in the context instead

Reply:
Sure, thanks for the catch.